include/tdep-riscv/jmpbuf.h

```diff
@@ -23,8 +23,6 @@ LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 
-#if defined __linux__
-
 /* https://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/riscv/setjmp.S;h=0b92016b311b11aa9eeb62b38c670a262f1924c9;hb=HEAD */
 #define JB_SP           13
 #define JB_RP           0
```

Comment:
This is specifically for Linux with glibc (should have added a `__GLIBC__` ifdef). FreeBSD has the following:

- https://github.com/freebsd/freebsd-src/blob/bada3811884cb22d088c8b8e4dbc39227682ce48/sys/riscv/include/setjmp.h#L40
- https://github.com/freebsd/freebsd-src/blob/bada3811884cb22d088c8b8e4dbc39227682ce48/lib/libc/riscv/gen/_setjmp.S#L42
- `man 3 setjmp`